.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Remove? To hide the fact that we're noobs and got away with good old copy-pasting? :)

Reply:
Done with 65b691f.